1159/en.md

```diff
@@ -0,0 +1,77 @@
+
+# LOJ 1159
+## Solution Approach:
+This code is for finding the length of the longest common subsequence (LCS) of three strings. 
+The LCS of three strings is the longest sequence of characters that appear in the same order in all three strings.
+
+The function batman takes in three strings, X, Y, and Z, as well as their lengths, and returns the length of the LCS of the three strings. 
+It does this by filling in a 3-dimensional array, L, using a bottom-up dynamic programming approach.
+
+The base case is when either the length of X, Y, or Z is 0, in which case the LCS is 0. If the last characters of X, Y, and Z are the same,
+then the LCS of the three strings is one more than the LCS of the three strings without the last characters. 
+Otherwise, the LCS is the maximum of the LCS of the three strings with X, Y, or Z shortened by one character. 
+# In short 
+-if the string matches then it goes diagoanlly(add 1 and increase i,j,k index)
+-if not then take max of neighbour elements
```

Comment:
This is just explaining what your code is doing. This won't fly.
Explain what the solution is, using sentences so that the reader doesn't need to resort to your source code to understand what's happening. 
Since this is a DP problem, you can start off by explaining what the states are, how computing the result for one state depends on multiple small states (in short, explain the sub-problems). 
As you mentioned LCS, it'd be good of you to provide a reference to an article about LCS as well; so that if the reader hasn't encountered LCS before, they can quickly jump to that article, read it and try the problem themselves again, before reading your editorial.

Reply:
Ok i am working on that